components/landing/faq.tsx

```diff
@@ -15,7 +15,11 @@ const FAQ = () => {
             id='accordion'
         >
             <span className='bold text-center text-2xl italic'>FAQ</span>
-            <Accordion type='single' collapsible className='w-96'>
+            <Accordion
```

Comment:
I still think having the big orange box with both parts is kind of weird and pointless. But I certainly think that you shouldn't have to scroll within the accordion to see all of the FAQ questions. I think it should be resized so that there is no inner scrolling.

Also, all of the questions need to be left aligned, the alignment is still messed up.